packages/vuetify/src/composables/defaults.ts

```diff
@@ -26,21 +26,24 @@ export function useDefaults () {
   return defaults
 }
 
-export function provideDefaults (props?: {
-  defaults?: DefaultsInstance
-  reset?: number | string
-  root?: boolean
-  scoped?: boolean
-}) {
-  const defaults = useDefaults()
+export function provideDefaults (
+  defaults?: Ref<DefaultsInstance | undefined> | DefaultsInstance,
+  options?: {
+    reset?: Ref<number | string | undefined>
+    root?: Ref<boolean | undefined>
+    scoped?: Ref<boolean | undefined>
+  }
+) {
+  const injectedDefaults = useDefaults()
+  const providedDefaults = wrapInRef(defaults)
```

Comment:
What is `wrapInRef` for? `ref(val)` already does nothing if `val` is already a Ref.